pymeasure/instruments/hp/hp437b.py

```diff
@@ -0,0 +1,949 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+from enum import Enum, IntFlag
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class MeasurementUnit(Enum):
+    """Enumeration to represent the measurement unit the power meter will measure in"""
+
+    Watts = 0
+
+    dBm = 1
+
+    Percent = 2
+
+    dB = 3
```

Comment:
Typically, enum members are written with all caps: "WATTS", "DBM", but you might leave them as they are.

Regarding the empty lines: I suggest you to remove them, as the members are all related.

```suggestion
    Watts = 0
    dBm = 1
    Percent = 2
    dB = 3
```